src/components/CopyToClipboard/CopyToClipboard.ts

```diff
@@ -72,14 +79,21 @@ export class CopyToClipboard extends ResultAction {
         if (navigator && navigator.clipboard && navigator.clipboard.writeText) {
             try {
                 await navigator.clipboard.writeText(text);
-                this.element.querySelector<HTMLElement>('.coveo-caption-for-icon').innerText = l('CopyToClipboard_copied');
             } catch (err) {
                 this.logger.error('Copy to clipboard failed.', text, err);
                 this.copyToClipboardFallback(text);
             }
         } else {
             this.copyToClipboardFallback(text);
         }
+        this.setToolipText(this.options.successTooltip);
+    }
+
+    private setToolipText(text: string) {
+        const tooltipElement = this.element.querySelector<HTMLElement>('.coveo-caption-for-icon');
+        if (tooltipElement && text) {
```

Comment:
What should be the behavior if I set `this.options.successTooltip=''` ?

Reply:
Je dirais de ne pas changer le texte

Reply:
Exact

Reply:
We need a test for that then :D